### ISSUE TITLE
websockets idea

### ISSUE THREAD

**Issue by gcmartijn** [omorandi/tiws#13]
Many users use socket.io or engine.io or just websockets (and there are more).
To tackle this problem and make our programs future proof I use this:
https://github.com/primus/primus

That is a global node.js server (and client) and you can add your own socket.

Then I thought how can I connect to my primus node.js server using android and ios using one framework (titanium) using websockets. I can't..

But what I can do is using Xcode to connect to primus using this:
https://github.com/daltoniam/Starscream/blob/master/WebSocket.swift with the websockets transport
and this uri: wss://hostname:8091/primus/?ns=e98f6d3a69e800563fff46a30997889075f2ddf8

In other words, could you check if its possible to connect to a simple primus server ?
using transport: websocket, socket.io, or engine.io
Just a simple connection , sending, receiving is enough.

After that, the users can create a primus server, choose a transport and use ios or android ! :)
